CMakeLists.txt

```diff
@@ -72,6 +74,11 @@ else ()
   if (NOT CALIPER_GIT_REPO)
     set(CALIPER_GIT_REPO https://github.com/LLNL/Caliper.git )
   endif()
+  if (NOT PYTHON_EXECUTABLE)
+    set(PYTHON_ADDITIONAL_VERSIONS "2.0 2.1 2.2 2.3 2.4 2.5 2.6 2.7")
```

Comment:
We probably don't need to support down to 2.0 (and it probably doesn't work earlier than 2.5 or 2.6) but I didn't have time to exhaustively check.